recipes-nymea/nymea-plugins-zigbee/nymea-plugins-zigbee_git.bb

```diff
@@ -0,0 +1,34 @@
+DESCRIPTION = "nymea-plugins-zigbee"
+
+LICENSE = "LGPL-3.0 | NYMEA-COMMERCIAL"
+LIC_FILES_CHKSUM="file://LICENSE.LGPL3;md5=3000208d539ec061b899bce1d9ce9404"
+
+SRC_URI="git://github.com/nymea/nymea-plugins-zigbee.git;protocol=https;branch=master"
+# Release: 1.0.0
+SRCREV="75314cf373476ee20a1545b834d4c66acdb6d851"
+PV = "git${SRCPV}"
+
+DEPENDS += "nymead nymead-native"
+
+inherit qmake5
+
+S = "${WORKDIR}/git"
+
+EXTRA_QMAKEVARS_PRE += "CONFIG+=selection ${PACKAGECONFIG_CONFARGS}"
```

Comment:
I think this is currently a no-op so can be removed.

Reply:
Oh, also, all the repositories have new build options now:
`WITH_PLUGINS="plugin1 plugin2 ..." `
would explicitly list plugins to be included (pretty much how the old `CONFIG+=selection SELECTION="plugin1 plugin2"` did and
`WITHOUT_PLUGINS="plugin1 plugin2 ..."`
would build all but the listed.

In any case, the "CONFIG+=selection" is no more...

Reply:
Some of the devices do not follow the zigbee specification and have a reverse engineered protocol. While they work perfectly fine and are very unlikely to stop working with a firmware update, it may still be desirable to disable them when building a product and only supply those who officially want to be connectable by 3rd party devices.

Reply:
Fixed and removed CONFIG+selection part.